src/transformers/modeling_tf_utils.py

```diff
@@ -365,9 +367,7 @@ def input_processing(func, config, input_ids, **kwargs):
                 if tensor_name in parameter_names:
                     output[tensor_name] = input
                 else:
-                    raise ValueError(
-                        f"The tensor named {input.name} does not belong to the authorized list of names {parameter_names}."
-                    )
```

Comment:
should we instead raise a warning here maybe?

Reply:
No need of an error or warning, as now, we allow tensors that have a different name. We don't have the choice because all the tensors that will be represented by a symbolic input simply cannot have a name that belongs to the parameters. So basically if you raise a warning everytime you will use a tf.data.dataset you will have this message raised at each batch, it will supply pollute your standard output for nothing.

Reply:
More specifically in the example given here https://github.com/huggingface/transformers/pull/8985#issuecomment-740654016

the symbolic inputs are:
```
input_ids = tf.keras.Input(shape=(128,), dtype='int32', name="input_ids")
attention_mask = tf.keras.Input(shape=(128, ), dtype='int32', name="attention_mask")
```

And the tensors that represents there symbolic inputs are respectively named `IteratorGetNext:0` and `IteratorGetNext:1` and these two names cannot be changed.